database/seeds/FakePlayerSeeder.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Database\Seeder;
+use App\Player;
+
+class FakePlayerSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        // Only run this if you're on a local environment
+        if(env('APP_ENV') === 'local') {
+            for ($x = 0; $x <= 3000; $x++) {
```

Comment:
Why so many?

Reply:
I've added code that looks for the max player id in the player_team table, then adds that many.